pom.xml

```diff
@@ -397,6 +397,10 @@
                     <groupId>io.swagger</groupId>
                     <artifactId>swagger-core</artifactId>
                 </exclusion>
+                <exclusion>
```

Comment:
@sfc-gh-lsembera Do you know what's the implication of excluding this in other dependencies? I assume we will use the version defined in the ingest sdk (or any other dependencies that uses this)? If we don't exclude this, we will just use a random version that appears to be the first one?

Reply:
And what's the best practice here? Should we define the version explicitly?

Reply:
If we define the dependency directly, there is no need to exclude it from other dependencies. Maven uses the [_nearest definition_](https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#transitive-dependencies) rule, which will pick the version declared directly.